### DISCUSSION TITLE
How can i intercept wrong cookies ?

### DISCUSSION THREAD

**Discussion by susanne99** [laravel/framework#52586] [General]
I use sanctum for authentication in Laravel 11. Works good. 
If I have my SPA-VUEJS app open in 2 separate browsers or separate browser tabs, how can I "cleanly" determine that one of the two APPs has logged out - i.e. the cookies are no longer valid. 
Of course, there is currently an error in the middleware server. 
I would have to be able to check, perhaps using Laravel middleware, whether the program still has valid login data and redirect to the login dialog

**Comment by ozdemirrulass** [comment]
Hi @susanne99,

You can find the detailed explanation here: [Laravel Sanctum SPA Authentication](https://laravel.com/docs/11.x/sanctum#spa-authentication).

If you have any further questions on the subject, please let me know, and I'll do my best to help.

**Comment by susanne99** [comment]
thank for your answer. i know the manual, an example would be nice.
so, at the end, the manual alone do not give me the solution - sorry.

**why isn't there more going on here - I'm here at Laravel, right?**
